src/lib/fetch.js

```diff
@@ -5,16 +5,15 @@
  * @module finder
  */
 
-const os = require('os');
-const path = require('path');
-const baseFetch = require('fetch-filecache-for-crawling');
+import os from 'node:os';
+import { cwd } from 'node:process';
```

Comment:
is that used anywhere?

Reply:
Nowhere. Good catch! :)

I had initially started loading JSON through dynamic imports and thought I'd need to resolve paths against the current working directory, but then I switched to a common `loadJSON` function (and the current working directory is the base folder for dynamic imports in any case)